examples/using-drupal/src/pages/index.js

```diff
@@ -185,8 +186,12 @@ class IndexPage extends React.Component {
                 flexWrap: `wrap`,
               }}
             >
-              {nextFourPromotedRecipes.map(recipe => (
-                <PromotedCard recipe={recipe} columns={2} />
+              {nextFourPromotedRecipes.map((recipe, index) => (
```

Comment:
Discard the `index` here and use `recipe.fields.slug` for the key.
It's unique: https://github.com/gatsbyjs/gatsby/blob/master/examples/using-drupal/gatsby-node.js#L7

Reply:
Thanks for the advice :)